### ISSUE TITLE
Volume SI Units are wrong

### ISSUE THREAD

**Issue by cheruvian** [harrydehix/simple-units#9]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `simple-units@1.0.5` for the project I'm working on.

tl;dr; SI volume units are wrong

```
units.default.from(1, 'ml').to('l')
```

results in

```
1e-9
```

which is incorrect (should be `0.001`). It looks like the issue is in the `siVolumePrefixes` which does an extra `.pow(3)`

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/simple-units/dist/units/variables/siVolumePrefixes.js b/node_modules/simple-units/dist/units/variables/siVolumePrefixes.js
index c097d80..62d285a 100644
--- a/node_modules/simple-units/dist/units/variables/siVolumePrefixes.js
+++ b/node_modules/simple-units/dist/units/variables/siVolumePrefixes.js
@@ -7,5 +7,5 @@ const siPrefixes_1 = __importDefault(require("./siPrefixes"));
 /**
  * {@link Variable} holding every SI volume prefix.
  */
-exports.default = siPrefixes_1.default.copy().pow(3);
+exports.default = siPrefixes_1.default.copy();
 //# sourceMappingURL=siVolumePrefixes.js.map
\ No newline at end of file
```

I'll try and find time to create a PR but hope this helps others

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by harrydehix** [comment]
Hey, thank you for telling me! Your fix unfortunately causes other unit conversions to not work properly. I have found another alternative that should work and fix the bug.
It's included in version 1.06.

**Comment by cheruvian** [comment]
Nice, I was actually only using it for volume and some custom stuff hah nice catch and thanks for the quick fix!
